src/onboarding/steps.ts

```diff
@@ -243,6 +264,34 @@ export function _getStepInfo({ firstScreenInStep, navigator, dispatch, props }:
           }
         },
       }
+    case Screens.SignInWithEmail:
+      return {
+        next: () => {
+          if (showRecoveryPhraseEducation) {
+            navigate(Screens.AccountKeyEducation, {
+              origin: 'cabOnboarding',
+            })
+          } else if (skipVerification || numberAlreadyVerifiedCentrally) {
+            dispatch(setHasSeenVerificationNux(true))
+            finishOnboarding(Screens.ChooseYourAdventure)
+          } else {
+            // DO NOT CLEAR NAVIGATION STACK HERE - breaks restore flow on initial app open in native-stack v6
+            navigate(Screens.VerificationStartScreen)
```

Comment:
In the videos, looks like after backup is successful, the flow navigates to home directly, is that expected? Per this, looks like it should go to phone verification / CYA?

Reply:
This is fixed in #5611.